### PR TITLE
chore(test-data): additional changes to dataset

### PR DESCRIPTION
**Pull request by jaikamat**
## Summary
- Prefer GB country code over UK
- Fix title of existing product preset
- Integrate UK zone into Europe zone

### CODE REVIEW

File: models/product/src/product/product-draft/presets/sample-data-goodstore/cobblestone-rug.spec.ts [hunk 1]
```diff
@@ -117,10 +117,10 @@ describe(`with cobblestoneRug preset`, () => {
         "metaTitle": undefined,
         "name": {
           "de": undefined,
-          "de-DE": "kopfsteinpflaster",
+          "de-DE": "Strukturteppich "Kopfsteinpflaster"",
           "en": undefined,
-          "en-GB": "cobblestone-rug",
-          "en-US": "cobblestone-rug",
+          "en-GB": "Cobblestone Rug",
```
Comment (jaikamat) [inline models/product/src/product/product-draft/presets/sample-data-goodstore/cobblestone-rug.spec.ts:122]: This was a rogue copypasta, the title of the product has now been fixed

File: models/cart/src/cart/cart-draft/presets/sample-data-goodstore/jennifer-jones-01.spec.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ describe('with the preset cart `jenniferJones01`', () => {
           "title": undefined,
         },
         "businessUnit": undefined,
-        "country": "UK",
+        "country": "GB",
```
Comment (jaikamat) [inline models/cart/src/cart/cart-draft/presets/sample-data-goodstore/jennifer-jones-01.spec.ts:40]: Changed all instances of country "UK" to "GB"

File: models/product/src/product/product-draft/presets/sample-data-goodstore/sunnai-glass-bowl.spec.ts [hunk 3]
```diff
@@ -4,7 +4,6 @@ import sunnaiGlassBowl from './sunnai-glass-bowl';
 describe(`with sunnaiGlassBowl preset`, () => {
   it(`should return a sunnaiGlassBowl preset`, () => {
     const sunnaiGlassBowlPreset = sunnaiGlassBowl().build<TProductDraft>();
-    console.log(sunnaiGlassBowlPreset);
```
Comment (jaikamat) [inline models/product/src/product/product-draft/presets/sample-data-goodstore/sunnai-glass-bowl.spec.ts:7]: removed some rogue console logs

File: models/zone/src/zone/zone-draft/presets/sample-data-goodstore/index.ts [hunk 4]
```diff
@@ -1,10 +1,8 @@
-import countryGermany from './country-germany';
-import countryUnitedKingdom from './country-unitedkingdom';
+import countryGermanyAndUk from './country-germany-and-uk';
```
Comment (jaikamat) [inline models/zone/src/zone/zone-draft/presets/sample-data-goodstore/index.ts:1]: renamed appropriately rather than `countryGermany`

**Review by stephsprinkle — APPROVED** [review]

**Pull request merged**